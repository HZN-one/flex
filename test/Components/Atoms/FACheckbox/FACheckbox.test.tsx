/**
 * @jest-environment jsdom
 */

import * as React from "react";
import "@testing-library/jest-dom";

import { FACheckbox } from "@Components";

import { render } from "@testing-library/react";

describe("FACheckbox", () => {
  it("should match snapshot", () => {
    const { container } = render(<FACheckbox testID="checkbox-example" />);
    expect(container).toMatchSnapshot();
  });
});
