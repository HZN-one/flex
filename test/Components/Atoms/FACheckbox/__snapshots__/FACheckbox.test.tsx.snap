// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FACheckbox should match snapshot 1`] = `
<div>
  <span
    class="MuiCheckbox-root MuiCheckbox-colorPrimary MuiButtonBase-root MuiCheckbox-root MuiCheckbox-colorPrimary PrivateSwitchBase-root css-12wnr2w-MuiButtonBase-root-MuiCheckbox-root"
    data-testid="checkbox-example"
  >
    <input
      class="PrivateSwitchBase-input css-1m9pwf3"
      data-indeterminate="false"
      type="checkbox"
    />
    <svg
      aria-hidden="true"
      class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
      data-testid="CheckBoxOutlineBlankIcon"
      focusable="false"
      viewBox="0 0 24 24"
    >
      <path
        d="M19 5v14H5V5h14m0-2H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2z"
      />
    </svg>
    <span
      class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
    />
  </span>
</div>
`;
